less/navigation.less

```diff
@@ -1 +1,67 @@
-// Navigation code here
\ No newline at end of file
+// Navigation code here
+
+.main-navigation {
+
+    .navmixin(@nav-bg, @nav-font);
+
+    display        : flex;
+    justify-content: center;
+    border         : 1px solid @sea-shell;
+
+    nav {
+        display       : flex;
+        flex-direction: row;
+
+        align-items   : baseline;
+        flex-direction: column;
+        width         : 800px;
+
+
+    }
+
+
+    h1 {
+
+
+        font-size : 4em;
+        align-self: center;
+        margin    : 2%;
+
+
+    }
+
+
+    ul {
+
+
+        display        : flex;
+        flex-direction : row;
+        align-self     : center;
+        justify-content: space-around;
+        width          : 450px;
```

Comment:
a fixed width isn't very responsive.  A solution I might offer is using a width using percentages and also tack on a min width
```
ul {
        width    : 40%;
        min-width: 30rem;
```